mllib/src/main/scala/org/apache/spark/ml/tree/treeModels.scala

```diff
@@ -58,3 +58,43 @@ trait DecisionTreeModel {
     header + rootNode.subtreeToString(2)
   }
 }
+
+/**
+ * :: AlphaComponent ::
+ *
+ * Abstraction for models which are ensembles of decision trees
+ *
+ * TODO: Add support for predicting probabilities and raw predictions
+ */
+@AlphaComponent
+trait TreeEnsembleModel {
```

Comment:
Should it be public? Note that adding method to an interface counts as a break change.